src/Simplex/Chat/Protocol.hs

```diff
@@ -335,8 +347,8 @@ data ChatMsgEvent (e :: MsgEncoding) where
   XGrpLinkInv :: GroupLinkInvitation -> ChatMsgEvent 'Json
   XGrpLinkReject :: GroupLinkRejection -> ChatMsgEvent 'Json
   XGrpLinkMem :: Profile -> ChatMsgEvent 'Json
-  XGrpLinkAcpt :: GroupMemberRole -> ChatMsgEvent 'Json
-  XGrpMemNew :: MemberInfo -> ChatMsgEvent 'Json
+  XGrpLinkAcpt :: GroupMemberRole -> Maybe MemberId -> ChatMsgEvent 'Json -- Maybe MemberId for compatibility
```

Comment:
compatibility with what? wasn't it just added for directory approvals?

Reply:
yes - but host (directory) sends it to invitee without MemberId now. It could be non optional otherwise